### DISCUSSION TITLE
React 19 Support

### DISCUSSION THREAD

**Discussion by dereckquock** [katspaugh/wavesurfer.js#3964] [Q&A]
Now that React 19 is stable, I want to use `wavesurfer-react`, but the `peerDependencies` need to be updated. Are there any plans to support this in the near future?

**Comment by katspaugh** [comment]
Should be an easy update, please feel free to make a PR if I don't get to it faster.

**Reply by dereckquock** [reply]
For sure, thanks!

**Reply by dereckquock** [reply]
i opened a PR for ya! https://github.com/katspaugh/wavesurfer-react/pull/10 🙌 thanks for takin' a look when you can 🙇‍♂️

**Reply by katspaugh** [reply]
Merged and released as 1.0.8, thank you!

**Reply by dereckquock** [reply]
amazing! thank you so much 💯
hope you have a nice holidays! 🎄
